### ISSUE TITLE
Horizontal and Vertical Stack card editor previews do not update

### ISSUE THREAD

**Issue by SeanPM5** [home-assistant/frontend#4537]
**Checklist:**
- [x] I updated to the latest version available
- [x] I cleared the cache of my browser

**Home Assistant release with the issue:**
0.104.2

**UI (States or Lovelace UI?):**
Lovelace

**Browser and Operating System:**
Chrome 79.0.3945.117 / macOS 10.13.6

**Description of problem:**
Preview on both Horizontal Stack and Vertical Stack cards doesn't ever update. There's no UI editor for these cards, but when you alter things in the code editor on the left, none of your changes will appear in the preview on the right.

Here you can see that the stuff in code editor is not the same as stuff in the preview.
![horizontal-stack](https://i.imgur.com/Qf8cdQM.png)

**Expected behaviour:**
Live update the preview

**Relevant config:**
```
type: horizontal-stack
title: Stuck Preview When Editing
cards:
  - type: entity-button
    tap_action:
      action: toggle
    name: Fake Light
    entity: light.fake
  - type: entity-button
    tap_action:
      action: toggle
    name: Fake Boolean
    entity: input_boolean.fake
```

**Steps to reproduce this problem:**
1. Copy my config from above, paste it into Horizontal Stack card editor
2. Now make any changes, like removing the second button or changing the title of the card. You'll notice the preview never updates.

**Comment by bramkragten** [comment]
Is it just the horizontal stack or both stack cards?

**Comment by iantrich** [comment]
If you're going to fix it, might as well finally figure out how we want to do a stack UI editor 😜

**Comment by bramkragten** [comment]
I have it figured out, just need you to build it 😜

**Comment by SeanPM5** [comment]
Just tested and it happens in vertical stack card as well, updated the title/OP.

**Comment by SeanPM5** [comment]
Appears to be fixed with new stack card editors in 0.107
